addons/website_sale/static/src/snippets/s_dynamic_category_snippet/000.scss

```
@@ -0,0 +1,30 @@
+.s_category_container{
```

```suggestion
.s_category_container {
```
Similar for other places